optaplanner-docs/src/main/asciidoc/MoveAndNeighborhoodSelection/MoveAndNeighborhoodSelection-chapter.adoc

```diff
@@ -1713,9 +1720,16 @@ Advanced configuration:
     <moveIteratorFactory>
       ... <!-- Normal moveSelector properties -->
       <moveIteratorFactoryClass>...</moveIteratorFactoryClass>
+      <moveIteratorFactoryCustomProperties>
+        ...<!-- Custom properties -->
+      </moveIteratorFactoryCustomProperties>
     </moveIteratorFactory>
 ----
 
+To configure values of an `MoveListFactory` dynamically in the solver configuration
```

Comment:
- an -> a
- the description is talking about a `MoveListFactory`, but the xml element is called `moveIteratorFactory`